src/Chip.js

```diff
@@ -1,23 +1,95 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-const Chip = ({ children, close, className, ...props }) => {
-  return (
-    <div className={cx('chip', className)} {...props}>
-      {children}
-      {close ? <i className="close material-icons">close</i> : null}
-    </div>
-  );
-};
+class Chip extends Component {
+  componentDidMount() {
+    const { options } = this.props;
+
+    if (options && M) {
+      this.instance = M.Chips.init(this._chips, options);
+    }
+  }
+
+  componentWillUnmount() {
+    this.instance && this.instance.destroy();
+  }
+
+  render() {
+    const { children, close, className, options } = this.props;
+    const classes = cx(
+      {
+        chip: !options,
+        chips: options
+      },
+      className
+    );
+    let chipContent = (
+      <React.Fragment>
+        {children}
+        {close ? <i className="close material-icons">close</i> : null}{' '}
```

Comment:
should we add here `<Icon>` component like in others?

Reply:
what's the empty string for?

Reply:
> what's the empty string for?

Whoops, my bad!